### PR TITLE
fix: update vendor-provider schema

### PR DESCRIPTION
**Pull request by riyaz-ali**
This PR addresses #828 and #843

### CODE REVIEW

File: migrations/900000000000042_add_comment_to_providers.up.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+-- SQL migration to add an optional comment field to providers
+BEGIN;
+
+ALTER TABLE mergestat.providers ADD COLUMN comment TEXT;
```
Comment (patrickdevivo) [inline migrations/900000000000042_add_comment_to_providers.up.sql:4]:
```suggestion
-- SQL migration to add an optional description field to providers
BEGIN;

ALTER TABLE mergestat.providers ADD COLUMN description TEXT;
```

Comment (patrickdevivo) [reply migrations/900000000000042_add_comment_to_providers.up.sql:4]: @riyaz-ali any reason for the comment vs description?

Comment (riyaz-ali) [reply migrations/900000000000042_add_comment_to_providers.up.sql:4]: no particular reason 😅 happy to rename it to `description` 👍

File: migrations/900000000000043_encrypt_username_in_crednetials.up.sql [hunk 2]
```diff
@@ -0,0 +1,35 @@
+-- SQL migration to update credential add and fetch routines to perform encryption and decryption
```
Comment (patrickdevivo) [inline migrations/900000000000043_encrypt_username_in_crednetials.up.sql:1]: just a nit - but see typo in file name `crednetials` @riyaz-ali

**Pull request merged**